core/src/services/webdav/backend.rs

```diff
@@ -322,18 +324,40 @@ impl Accessor for WebdavBackend {
             return Ok(RpStat::new(Metadata::new(EntryMode::DIR)));
         }
 
-        let resp = self.webdav_head(path).await?;
+        let mut header_map = HeaderMap::new();
+        // not include children
+        header_map.insert("Depth", "0".parse().unwrap());
+        header_map.insert(
+            header::ACCEPT,
+            "text/plain,application/xml".parse().unwrap(),
```

Comment:
**request-change**: accept `text/plain` is wrong as we can't parse plain response.